src/main/java/geometry/Square.java

```diff
@@ -0,0 +1,15 @@
+package geometry;
+
+public class Square extends Rectangular {
+
+    public Square(double sideA) {
+        // add default sideB = 0, because we really do not need sideB for Square
+        super(sideA, 0);
+    }
+
+    @Override
+    public double getSquare() {
+        return Math.pow(this.getSideA(), 2);
+    }
+
```

Comment:
please do comments mentioned above for the Circle class as well

Reply:
done